src/SourceLink.GitWeb/build/SourceLink.GitWeb.targets

```diff
@@ -0,0 +1,65 @@
+<?xml version="1.0" encoding="utf-8"?>
+<Project>
```

Comment:
The file name is missing `Microsoft.` prefix. That's probably why the build is failing.

Reply:
Whoops. Good catch. Guess I missed a couple when I decided to go back to the Microsoft namespace.
Done.